### ISSUE TITLE
feat: send an admin email for failed and abandoned donations.

### ISSUE THREAD

**Issue by samsmith89** [impress-org/givewp#3877]
# Feature Request

## User Story

As a site admin, I want to be alerted by email when failed and abandoned donations happen on my site so that I can intervene early to resolve issues.

This would be important for me, because if donations are failing at Give, and not making it to the payment gateway the only way I'll notice is if I sign into the backend of my site and see them. Being alerted early gives me the chance to resolve the issue before I lose out on too many donations.

It may also be beneficial to have this email fire only after a certain number of failed or abandoned donations in a row so I don't receive an email everytime a card is declined.

## Related

https://secure.helpscout.net/conversation/718234863/29984

## Acceptance Criteria

- [ ] I am somehow alerted when an overwhelming amount of abandoned or failed donations are coming into my site because of a resolvable issue.

**Comment by kevinwhoffman** [comment]
I agree with limiting the number of core emails, however I think this could be a valuable email. The ability to recover even a single large donation could mean a lot to an organization. The newly organized email tabs that separate admin and donor emails will greatly improve the admin's understanding of the emails and their purpose, which makes me more inclined to allow this one.

@mathetos had also mentioned in Slack that he did not want to get too deep into cart recovery territory, but that this email could be a small addition that satisfies that need for customers looking for that type of functionality.

**Comment by mathetos** [comment]
@DevinWalker Regarding the Zapier route, I'm not 100% certain that would even work at this time because of this existing issue with Zapier:
https://github.com/impress-org/give-zapier/issues/19

If we knew with certainty that both Pending and Failed statuses sent appropriate actions to Zapier reliably then that might suffice completely. But I believe there's some trouble with that currently per the issue above.

**Comment by kevinwhoffman** [comment]
@DevinWalker Can you please make a product decision on this one so we can either close as wontfix or move forward with development of the email notification? Thanks.

**Comment by kevinwhoffman** [comment]
Closing. The Zapier issue https://github.com/impress-org/give-zapier/issues/19 has since been resolved, so triggering an email for failed donations should now be possible with Zapier if needed.
